src/helpers/getCurrenciesFromDomain.ts

```diff
@@ -0,0 +1,38 @@
+import { Asset, StellarTomlResolver } from "stellar-sdk";
```

Comment:
i didn't know stellar-sdk had a toml resolver! I wrote my own last night because I'm doing something similar in Freighter 😭 😆

Reply:
Oh no...